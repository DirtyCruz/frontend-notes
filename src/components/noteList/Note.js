import React from 'react';
import PropTypes from 'prop-types';

function Note({ title, body }) {
  return (
    <>
      <h4>{title}</h4>
      <p>{body}</p>
    </>
  );
}

Note.propTypes = {
  title: PropTypes.string.isRequired,
  body: PropTypes.string.isRequired
};

export default Note;
