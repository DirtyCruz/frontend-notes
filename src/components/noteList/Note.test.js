import React from 'react';
import { shallow } from 'enzyme';
import Note from './Note';

describe('Note component', () => {
  it('renders Note', () => {
    const wrapper = shallow(<Note title='title' body='body' />);
    expect(wrapper).toMatchSnapshot();
  });
});
