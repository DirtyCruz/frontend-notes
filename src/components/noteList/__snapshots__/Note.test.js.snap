// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Note component renders Note 1`] = `
<Fragment>
  <h4>
    title
  </h4>
  <p>
    body
  </p>
</Fragment>
`;
